### DISCUSSION TITLE
Export settings and aliases from subscription to self hosted

### DISCUSSION THREAD

**Discussion by eTaurus** [simple-login/app#609] [Q&A] [answered]
Hi!
I would like to start using this fantastic project. My plan is to self host it on my Raspberry Pi 4 with 8 GB RAM but not until there is a working solution for **docker-compose**.
In the meantime I would like to subscribe to your service, maybe even pay for Premium until I can self host it (and after that become a patron on Patreon).

I wonder if it is possible to export the complete configuration from the subscription to my self hosted solution when it is time to migrate.

**Comment by nguyenkims** [comment, marked as answer]
We have a work-in-progress PR to add docker-compose for SimpleLogin at https://github.com/simple-login/app/pull/355 but it's been in pause for a while. I think there are other people who have successfully run SimpleLogin within docker-compose so maybe you can use their `docker-compose.yml` file in the meantime.

**Reply by eTaurus** [reply]
Thank you for your reply. It would be great if someone with a working `docker-compose.yml` would be so kind to share it with me. 

What about the other part of my question? Is it possible to just export everything and migrate to self hosted?

**Reply by nguyenkims** [reply]
Ah sorry I missed this question! 

You can export all your data from the SaaS solution and import it into your self-hosted version. Please note that the import can only work with your custom domain and not the domains provided by SimpleLogin.

**Reply by eTaurus** [reply]
Thanks!
